### DISCUSSION TITLE
Welcome to Leetcode-Premium-Unlocker Discussions!

### DISCUSSION THREAD

**Discussion by Edwardsoen** [Edwardsoen/Leetcode-Premium-Unlocker#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Anubhab2003** [comment]
It would be really helpful if we could access the Video Solution using the extension.
All though all the other premium feature are accessible.
Would like to Appreciate the work and effort Developers have put in.
Hope a positive Response.
